### ISSUE TITLE
Space Bar does not work on Mac when Full Keyboard Access is turned on

### ISSUE THREAD

**Issue by mikeritter** [warpdotdev/Warp#3934]
### Describe the bug

When I activate Accessibility > Keyboard > Full Keyboard Access there is a blue box from the previous overlaying the Warp window and space bar does not work to create a space.

### To reproduce

1) Open Warp
2) Open System Settings 
3) Select "Accessibility"
4) Select "Keyboard"
5) Toggle on "Full Keyboard Access"
6) Switch back to Warp
7) Type man, space bar, ls

### Operating system

MacOS

### Operating system and version

14.1.2

### Current Warp version

v0.2023.11.28.08.02.stable_00

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by heijligers** [comment]
same issue here

**Comment by epmartini** [comment]
👍

**Comment by Ash-Stuart** [comment]
+1

**Comment by chanv64** [comment]
hi, I just encountered this issue on my iMac-air when Full Keyboard access is turned on. Turning this off will make my spacebar works as per normal. I found out also that this issue (having Full Keyboard access turned on disable spacebar) doesn't work on other terminal apps like kitty, Alacritty, wezterm. Surprising, it works on iTerm2. I was scratching my head as to why my spacebar doesn't work until I came across this post. Just to inform that this is not warp terminal specific. As to why iTerm2 is not affected, I have no clue.

**Comment by cramhead** [comment]
I've encountered the same problem with Sequoia 15.0.1. I was completely baffled by what happened when I first noticed the issue in the Zed editor. When I found the same issue in Warp I speculated that the issue may be in a Rust dependency they share. Luckily this issue came up when I searched.

**Comment by dannyneira** [comment]
Hi Folks, we'll be tracking this issue here https://github.com/warpdotdev/Warp/issues/2734 and would appreciate it if you could 👍 and comment there if you're also experiencing the same bug. This helps us gauge the impact of the issue. 

We'll be closing this as a duplicate.

**Comment by dannyneira** [comment]
Duplicate of https://github.com/warpdotdev/Warp/issues/2734
